target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8068-cryptid-common.dtsi

```diff
@@ -0,0 +1,241 @@
+// SPDX-License-Identifier: GPL-2.0 OR MIT
+
+#include "qcom-ipq8064-v2.0.dtsi"
+
+#include <dt-bindings/soc/qcom,tcsr.h>
```

Comment:
Is this required?
I could not find `TCSR_USB_SELECT_USB3_x` or `IPQ806X_TCSR_REG_x_ADM_MUX_SEL` in dts(i).

Reply:
Had mistaken this for `qcom,gsbi.h`, which is already included in ipq8064.dtsi. This has been removed, likewise from the mr42/mr52 dts files. Thanks!